src/main/java/net/dv8tion/jda/api/entities/ThreadChannel.java

```diff
@@ -473,6 +473,13 @@ default ThreadMember getOwnerThreadMember()
     @Nonnull
     AutoArchiveDuration getAutoArchiveDuration();
 
+    /**
+     * The timestamp when this thread was created.
+     *
+     * @return The timestamp when this thread was created
+     */
```

Comment:
These docs need to mention that this will only be valid for ThreadChannels created after 2022-01-09. Otherwise, it will default to using the id of the thread which if it is based on a root message will be the creation time of the message, not the thread.